scripts/install-newrelic-alpine.sh

```diff
@@ -46,8 +46,9 @@ mkdir -p /var/log/newrelic
 chmod 777 /var/log/newrelic
 cp agent/scripts/newrelic.ini.template "${CONF_PHPMODS}"/newrelic.ini
 
+# Delete compilation directories, no longer needed
+go clean --cache
```

Comment:
@bryanlatten : More of a question than anything (and its probably small anyway) but I didn't notice `go` being installed in `docker-nginx`. Perhaps not part of this PR but do we keep `go` around or we purge it later on if it's not needed?

Reply:
@adobejmong `go` isn't installed in docker-nginx? Surrounding this script installs and removes it. Note: the first version of this script had all in-script dependencies installed/removed together, but on removal it would also wipe out dependencies that were still needed...so we're tracking them all at the top level.

Reply:
Perfect. Thanks for clarifying.